.travis.yml

```diff
@@ -1,30 +1,45 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+sudo: false
 
-env:
-  - SYMFONY_VERSION=2.1.*
-  - SYMFONY_VERSION=2.2.*
-  - SYMFONY_VERSION=2.3.*
-  - SYMFONY_VERSION=dev-master
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+    - php: 5.3
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.7.*
+    - php: 5.4
+      env:
+        - deps=high
+        - SYMFONY_VERSION=2.7.*
+    - php: 5.5
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.8.*
+    - php: 5.6
+      env:
+        - deps=high
+        - SYMFONY_VERSION=3.0.*
+    - php: 7.0
+      env:
+        - deps=low
+        - SYMFONY_VERSION=3.1.*
+    - php: 7.1
+      env:
+        - deps=high
+        - SYMFONY_VERSION=3.1.*
```

Comment:
Do we actually need to test each version one by one? Imo as both php and symfony has a bc policy, we can only test the lowest and highest version of each major (php 5.3/sf 2.7/sf 3.0, php 7.1/sf 2.*/sf 3.*).